build.sbt

```diff
@@ -1,65 +1,59 @@
 import sbt.Keys._
 import ReleaseTransformations._
 import Dependencies._
-import sbtrelease.{Version, versionFormatError}
+import sbtversionpolicy.withsbtrelease.ReleaseVersion
 
-/* --------------------------------------------------------------------- */
-
-val snapshotReleaseType = "snapshot"
-val snapshotReleaseSuffix = "-SNAPSHOT"
-
-lazy val versionSettingsMaybe = {
-  sys.props.get("RELEASE_TYPE").map {
-    case v if v == snapshotReleaseType => snapshotReleaseSuffix
-    case _ => ""
-  }.map { suffix =>
-    releaseVersion := {
-      ver => Version(ver).map(_.withoutQualifier.string).map(_.concat(suffix)).getOrElse(versionFormatError(ver))
-    }
-  }.toSeq
-}
+licenses := Seq(License.Apache2)
+organization := "com.gu"
+organizationName := "Guardian News & Media Ltd"
+organizationHomepage := Some(url("https://www.theguardian.com/"))
+val ghProject = "content-api-client"
 
 lazy val root = (project in file("."))
   .aggregate(client, defaultClient)
-  .settings(commonSettings, versionSettingsMaybe, publishSettings)
+  .settings(commonSettings)
   .settings(
-    publish / skip     := true,
-    releaseVersionFile := baseDirectory.value / "version.sbt",
-    Compile / sources  := Seq.empty,
-    Test / sources     := Seq.empty
+    publish / skip := true,
+    releaseVersion := ReleaseVersion.fromAggregatedAssessedCompatibilityWithLatestRelease().value,
+    releaseProcess := Seq(
+      checkSnapshotDependencies,
+      inquireVersions,
+      runClean,
+      setReleaseVersion,
+      commitReleaseVersion,
+      tagRelease,
+      setNextVersion,
+      commitNextVersion
+    )
   )
 
 lazy val client = (project in file("client"))
-  .settings(commonSettings, clientSettings, publishSettings)
+  .settings(commonSettings, clientSettings)
   .enablePlugins(BuildInfoPlugin)
 
 lazy val defaultClient = (project in file("client-default"))
   .dependsOn(client)
-  .settings(commonSettings, defaultClientSettings, publishSettings)
+  .settings(commonSettings, defaultClientSettings)
 
-/* --------------------------------------------------------------------- */
 
-lazy val commonSettings: Seq[Setting[_]] = Metadata.settings ++ Seq(
+lazy val commonSettings: Seq[Setting[_]] = Seq(
   crossScalaVersions      := scalaVersions,
   scalaVersion            := scalaVersions.max,
-  javacOptions            ++= Seq("-source", "1.8", "-target", "1.8"),
-  scalacOptions           ++= Seq("-deprecation", "-unchecked"),
+  scalacOptions           ++= Seq("-deprecation", "-unchecked", "-release:8"),
```

Comment:
Using `-release:8` will include setting `"-target", "1.8"` for the Java compiler, and we don't need `"-source", "1.8"` as we are not using any Java source files!